your_child_workflow/your_child_workflow_dacx.py

```diff
@@ -0,0 +1,55 @@
+from temporalio import workflow
+from activities import ComposeGreetingInput
+from temporalio.workflow import ParentClosePolicy
+
+"""dacx
+To spawn a Child Workflow Execution in Python, use the [`execute_child_workflow()`](https://python.temporal.io/temporalio.workflow.html#execute_child_workflow) function which starts the Child Workflow and waits for completion or
+use the [`start_child_workflow()`](https://python.temporal.io/temporalio.workflow.html#start_child_workflow) function to start a Child Workflow and return its handle. 
+This is useful if you want to do something after it has only started, or to get the Workflow/Run ID, or to be able to signal it while running.
+
+:::note
+
+`execute_child_workflow()` is a helper function for `start_child_workflow()` plus `await handle`.
+
+:::
+dacx"""
+
+"""dacx
+Set the `parent_close_policy` parameter inside the [`start_child_workflow`](https://python.temporal.io/temporalio.workflow.html#start_child_workflow) function or the [`execute_child_workflow()`](https://python.temporal.io/temporalio.workflow.html#execute_child_workflow) function to specify the behavior of the Child Workflow when the Parent Workflow closes.
+dacx"""
+
+
+@workflow.defn
+class ComposeGreeting:
```

Comment:
Sometimes you suffix workflows with `Workflow` sometimes not

Reply:
:+1: We should fix both (and anywhere we are inconsistent right next to each other)